streams/src/main/java/org/apache/kafka/streams/kstream/ValueTransformer.java

```diff
@@ -20,7 +20,7 @@
 import org.apache.kafka.streams.processor.ProcessorContext;
 
 /**
- * A stateful Value Transformer interface for transform a value into a new value.
+ * A stateful {@link ValueTransformer} interface for transform a value into a new value.
```

Comment:
What about:

> Defines a stateful transformation of a record value into a new value.

(Not sure whether this wording approach is consistent with other interface docstrings.)